gateway/1.8/modules/ROOT/pages/flex-architecture-high-availibility.adoc

```diff
@@ -0,0 +1,66 @@
+= Flex Gateway Disaster Recovery
```

Comment:
There's a typo in this adoc file name. It should be:
`flex-architecture-high-availability.adoc`.

Customers can see the filenames in the URL in the browser, so it'd be good to not have typos in the name! :)